pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -1197,7 +1197,7 @@ one = 'ID of the Kafka instance owner'
 
 [kafka.create.flag.clusterId.description]
 description = 'Description for the --cluster-id flag'
-one = 'ID of the Openshift Cluster data plane to create the Kafka instance on.'
+one = 'ID of the Openshift Cluster data plane to create the Kafka instance on'
```

Comment:
this change should probably be on its own PR

Reply:
Pushed changes here to make the checks pass, creating separate PR.

Reply:
Raised follow up at #1846